node/cli/src/lib.rs

```diff
@@ -113,6 +111,13 @@ pub fn get_service_config() -> service::ServiceConfig {
               .help("Sets public key to sign with, \
                          can be omitted with 1 file in keystore")
               .takes_value(true)
+        ).arg(
+            Arg::with_name("prod_block_on_tx")
```

Comment:
--immediate

Reply:
also can revert, and make it --batch_transactions to batch them and by default it's immediate.